pkg/cli/token/token.go

```diff
@@ -139,6 +145,51 @@ func generate(app *cli.Context, cfg *cmds.Token) error {
 	return nil
 }
 
+func Rotate(app *cli.Context) error {
+	if err := cmds.InitLogging(); err != nil {
+		return err
+	}
+	fmt.Println("\033[33mWARNING\033[0m: Recommended to keep a record of the old token. If restoring from a snapshot, you must use the token associated with that snapshot.")
```

Comment:
I am slightly curious what these escape codes will look like when rancher-system-agent pulls the logs, but it's probably fine.

Reply:
Yeah IDK, it might just be the escape characters, probably the same as the existing logrus.Warn output. PSA, kubectl doesn't use ASCII color codes and [people have been asking for support for a while](https://github.com/kubernetes/kubectl/issues/524).